src/sonic-yang-models/yang-models/sonic-fine-grained-ecmp.yang

```diff
@@ -72,22 +97,27 @@ module sonic-fine-grained-ecmp {
                 leaf bucket_size{
                     type uint16;
                     mandatory true;
-                    description "total hash bucket size desired,
-                    recommended value of Lowest Common Multiple
-                    of 1..{max # of next-hops}";
+                    description "total hash bucket size desired, recommended value of Lowest Common Multiple of 1..{max # of next-hops}";
                 }
 
                 leaf match_mode{
                     type match-mode-name;
                     mandatory true;
-                    description " The filtering method used to identify
-                    when to use Fine Grained vs regular route handling.
-                    nexthop-based looks to next-hop IP to filter routes
-                    and uses fine grained ecmp when nexthop IPs matches
-                    FG_NHG_MEMBER IPs. route-based looks to prefix to
-                    filter routes, and uses fine grained ecmp when the
-                    route prefix matches the FG_NHG_PREFIX prefix.";
+                    description " The filtering method used to identify when to use Fine Grained vs regular route handling. nexthop-based looks to next-hop IP to filter routes and uses fine grained ecmp when nexthop IPs matches FG_NHG_MEMBER IPs. route-based looks to prefix to filter routes, and uses fine grained ecmp when the route prefix matches the FG_NHG_PREFIX prefix.";
```

Comment:
update description to include prefix-based?

Reply:
Updated description.